src/components/Book.js

```diff
@@ -8,13 +8,25 @@ color: white;
 text-align: center;
 width: 100%;
 `;
+const BUTTON = styled.button`
+  border: 1px solid gray;
+  border-radius: 12px;
+  padding: 13px;
+  outline:none;
+  cursor:pointer;
+`;
```

Comment:
[**Optional**]
Wow, this is great.
I think it would be nice and professional to separate your `CSS` from the logic files, right?